pkg/awsclient/client.go

```diff
@@ -212,22 +221,40 @@ func (c *AwsClient) DescribeLoadBalancers(i *elb.DescribeLoadBalancersInput) (*e
 	return c.elbClient.DescribeLoadBalancers(i)
 }
 
+func (c *AwsClient) DescribeLoadBalancersV2(i *elbv2.DescribeLoadBalancersInput) (*elbv2.DescribeLoadBalancersOutput, error) {
+	return c.elbv2Client.DescribeLoadBalancers(i)
+}
+
+func (c *AwsClient) DeleteLoadBalancerV2(i *elbv2.DeleteLoadBalancerInput) (*elbv2.DeleteLoadBalancerOutput, error) {
+	return c.elbv2Client.DeleteLoadBalancer(i)
+}
+
+func (c *AwsClient) CreateLoadBalancerV2(i *elbv2.CreateLoadBalancerInput) (*elbv2.CreateLoadBalancerOutput, error) {
```

Comment:
Why are we adding this V2 suffix? Is it to avoid collisions with the version that Lisa is working on?

Reply:
There are 2 versions of the elb API, but the function names are the same from the aws sdk

Reply:
V2 is specifically for non-classic ELB functions. Yes it is to avoid confusion as classic elb could also have a function called CreateLoadBalancer